src/main/steps/applicant1/hub-page/joint/form.njk

```diff
@@ -24,5 +24,14 @@
       value: "Yes",
       preventDoubleClick: true
     }) }}
+  {% elif displayState.state() == 'AwaitingFinalOrder' %}
+      {{ govukButton({
+        text: applyForFinalOrder,
+        classes: form.submit.classes,
+        type: "submit",
+        name: "TODO",
```

Comment:
could add a todo comment next to this to highlight it

Reply:
Will do

Reply:
Can't add todo to njk